lib/matplotlib/axes/_base.py

```diff
@@ -2402,14 +2402,14 @@ def autoscale_view(self, tight=None, scalex=True, scaley=True):
                 (self._xmargin and scalex and self._autoscaleXon) or
                 (self._ymargin and scaley and self._autoscaleYon)):
             stickies = [artist.sticky_edges for artist in self.get_children()]
-            x_stickies = np.array([x for sticky in stickies for x in sticky.x])
-            y_stickies = np.array([y for sticky in stickies for y in sticky.y])
+            x_stickies = np.sort([x for sticky in stickies for x in sticky.x])
+            y_stickies = np.sort([y for sticky in stickies for y in sticky.y])
             if self.get_xscale().lower() == 'log':
                 x_stickies = x_stickies[x_stickies > 0]
             if self.get_yscale().lower() == 'log':
                 y_stickies = y_stickies[y_stickies > 0]
         else:  # Small optimization.
-            x_stickies, y_stickies = [], []
+            x_stickies = y_stickies = np.array([])
```

Comment:
Is it a problem that these are now the same object? I assume we do not mutate these values below?

Reply:
Technically, this is probably working. However, I'd prefer
~~~
x_stickies = np.array([])
y_stickies = np.array([])
~~~
First, we don't know if these might be written to for some reason in the future. Second, it keeps the symmetry.

Reply:
I moved the conditional higher to avoid this need.